doc/source/ray-core/examples/highly_parallel.ipynb

```diff
@@ -0,0 +1,363 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "515dffba",
+   "metadata": {},
+   "source": [
+    "# Using Ray for Highly Parallelizable Tasks\n",
+    "\n",
+    "While Ray can be used for very complex parallelization tasks,\n",
+    "often we just want to do something simple in parallel.\n",
+    "For example, we may have 100,000 time series to process with exactly the same algorithm,\n",
+    "and each one takes a minute of processing.\n",
+    "\n",
+    "Clearly running it on a single processor is prohibitive: this would take 70 days.\n",
+    "Even if we managed to use 8 processors on a single machine,\n",
+    "that would bring it down to 9 days. But if we can use 8 machines, each with 16 cores,\n",
+    "it can be done in about 12 hours.\n",
+    "\n",
+    "How can we use Ray for these types of task? \n",
+    "\n",
+    "We take the simple example of computing the digits of pi.\n",
```

Comment:
It seems like the script requires a large-scale cluster to reproduce the result. Should we have a link to the ray cluster guidance? https://docs.ray.io/en/master/cluster/quickstart.html

Reply:
Fixed. Added the following notice to the top of the document: 

The following code (and this notebook) assumes you have already set up your Ray cluster and that you are running on the head node. For more details on how to set up a Ray cluster please see the [Ray Cluster Quickstart Guide](https://docs.ray.io/en/master/cluster/quickstart.html).